File: 문자열_다루기_기본.py
def solution(s):
    if len(s) != 4 and len(s) != 6:
        return False
    else:
        only_digit = True
        for i in range(len(s)):
            if not s[i].isdigit():
                only_digit = False
                
        return only_digit

File: 문자열_다루기_기본.cpp
#include <iostream>
using namespace std;

bool solution(string s) {
	if (s.length() != 4 && s.length() != 6) return false;
	else {
		bool only_digit = true;
		for (int i = 0; i < s.length(); i++)
			if (!isdigit(s[i])) only_digit = false;

		return only_digit;
	}
}
